pkgs/by-name/wc/wcslib/package.nix

```diff
@@ -23,6 +23,10 @@ stdenv.mkDerivation rec {
     "man"
   ];
 
+  postInstall = ''
+    rm $out/share/doc/wcslib/wcslib
```

Comment:
I know this fixes the build, but nuking paths without any apparent connection to the underlying issue gives me a bad feeling. We have no way of detecting when this workaround can be removed again.

Reply:
Issue raised in the original PR: https://github.com/NixOS/nixpkgs/pull/370750#issuecomment-2646326423

Reply:
No matter how that discussion goes, I think this PR at least needs a comment in the code that references "noBrokenSymlinks" or something that.